README.md

```diff
@@ -6,6 +6,51 @@
 
 A megabot for everything CSCH needs, composed of smaller mostly standalone packages. This project uses [Discord.js](https://discord.js.org/#/).
 
+## Commands
+
+**Default prefix:** `++`
+
+**Command types:**
+
+|    Type    |                                                  Description                                                  |
+| ---------- | ------------------------------------------------------------------------------------------------------------- |
+|   `ama`    |                              Used to queue or collect questions for AMA events.                               |
```

Comment:
this is more expansive than just questions for events, but a lot of it is still in dev

Reply:
I took the descriptions for each command category from /specs, so I'll update it in both the table and in the spec sheet.